dxr/filters.py

```diff
@@ -233,4 +235,4 @@ def _should_be_highlit(self, entity):
         """
         return ((not self._term['qualified'] and
                  super(QualifiedNameFilterBase, self)._should_be_highlit(entity))
-                or entity['qualname'] == self._term['arg'])
+                or is_in(self._term['arg'], entity['qualname']))
```

Comment:
I like that we're able to implement this feature without adding another field to the doc.